shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -627,6 +628,48 @@ - (void)testItReportsHighContrastWhenTraitCollectionRequestsIt {
   [mockTraitCollection stopMocking];
 }
 
+- (void)testItReportsNoAccessibilityFlagsSet {
+  if (@available(iOS 13, *)) {
+    // noop
+  } else {
+    return;
+  }
+
+  // Setup test.
+  FlutterViewController* viewController =
+      [[FlutterViewController alloc] initWithEngine:self.mockEngine nibName:nil bundle:nil];
+
+  id valueProvider = OCMClassMock([AccessibilityValueProvider class]);
+  OCMStub([valueProvider accessibilityIsOnOffSwitchLabelsEnabled]).andReturn(false);
+
+  // Exercise behavior under test.
+  int32_t flags = [viewController accessibilityFlags];
+
+  // Verify behavior.
+  XCTAssert(flags == 0);
```

Comment:
Just in case the test device has some other `UIAccessibility` features on can you instead just check that the `kOnOffSwitchLabels` bit is off?